flake.nix

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Please remove the Flake files.

It's not your fault though; just that I don't use Nix and I am a bit cautious of adding files that I don't understand to the repository...

Reply:
Sure, no problem.